meps/mep-cross-references.md

```diff
@@ -0,0 +1,339 @@
+| Target in a document     | `<project:target.md#file>` | `[](file.md#target)`      |
+| Target in project        | `<project:#target>`        | `[](#target)`             |
+
+### Search Order and Specificity
+
+All references search the local document first[^specific_doc], then the local project in the order of the table of contents. A `xref_multiple` warning is raised if multiple matches are found.
+
+[^specific_doc]: With the exception of an explicit reference to a specific page, i.e. `[](./examples/my-doc.md#explicit-reference)`
+
+In large documentation sites, a referenced target can be present in multiple documents, in that case, the parser will emit a `xref_multiple` warning letting you know that there are multiple matches for the intended target.
+
+### Implicit Section Headers
+
+We suggest a configuration option to create anchor "slugs" for section headers, which stay close to the [GitHub implementation](https://github.com/Flet/github-slugger), which produces references that:
+
+- lower-case text
+- remove punctuation
+- replace spaces with `-`
+- enforce uniqueness via suffix enumeration `-1`
+
+For example, `## Links and Referencing` can be referenced as `[](#links-and-referencing)`.
+These are **implicit** references, and referring to them should raise an `xref_implicit` warning, which can optionally be suppressed by users.
+
+Implicit references are **not** available project wide, and are only accessible in the current document, as many documents follow similar structures (Abstract, Introduction, Methods, Summary). Adding two sections of the same name does not raise a duplicate identifier warnings (`xref_duplicate`), section identifiers are only unique to the document.
+
+### Paths
+
+- Links can be relative from the containing file.
+- Links can be absolute paths from the project root.
+- Absolute paths start with `/`.
+- The path separator is POSIX `/`.
+- The path must include the extension.
+
+### Downloads
+
+Files that are outside of the table of contents of the project and are referenced directly are downloads.
+
+- These follow the same path rules as above when referring to a unknown file-type (e.g. `./my-file.txt`).
+- Files with known file extensions (e.g. `*.md`, `*.ipynb`) will default to being document links, not downloads.
+- Downloads can be specified explicitly by a `<path:my-file.md>`, which will revert to a download regardless of the extension. (i.e. this overrides the default for `[](my-file.md)`, which is `<project:my-file.md>`).
+
+### Warnings and Errors
+
+`xref_missing`
+: There is a missing reference, that could not be found.
+
+`xref_implicit`
+: You are referencing an implicit reference which could change easily in the future, consider making this explicit.
+
+`xref_unsupported`
+: Raised if the the current environment does not support the reference look up. For example, single page builds.
+
+`xref_multiple`
+: Raised when multiple conflicting targets are matched.
+
+`xref_duplicate`
```

Comment:
Does this need to be separate from the missing/ambiguos warning?

Reply:
Removed.